packages/graphql_codegen/pubspec.yaml

```diff
@@ -8,23 +8,23 @@ homepage: https://github.com/heftapp/graphql_codegen/tree/main/packages/graphql_
 repository: https://github.com/heftapp/graphql_codegen/tree/main/packages/graphql_codegen
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
   gql: ^0.13.0
-  build: ^2.0.1
-  glob: ^2.0.1
-  code_builder: ^4.0.0
-  dart_style: ^2.0.1
-  built_collection: ^5.0.0
+  build: ^2.2.1
+  glob: ^2.0.2
+  code_builder: ^4.1.0
+  dart_style: ^2.1.1
+  built_collection: ^5.1.1
   path: ^1.8.0
-  gql_code_builder: ^0.2.0
+  gql_code_builder: ^0.3.0-alpha.0
```

Comment:
This dependency needs to be in sync with `graphql` and `graphql_flutter`. Furthermore, I would love if we can try not to depend on pre-releases.